aks-linux-sample/attestation-client.Dockerfile

```diff
@@ -10,15 +10,20 @@ RUN apt-get  install -y \
     cmake \
     wget \
     git
-RUN apt-get install -y  jq
 
 RUN wget https://packages.microsoft.com/repos/azurecore/pool/main/a/azguestattestation1/azguestattestation1_1.0.2_amd64.deb
```

Comment:
Please update to the latest: azguestattestation1_1.0.5_amd64.deb

Reply:
Done